.github/workflows/release.yml

```
@@ -27,6 +27,19 @@ jobs:
             false
           fi
 
+  set-tags:
+    runs-on: ubuntu-latest
+    outputs:
+      is_release: ${{ steps.check-tag.outputs.is_release }}
+    steps:
+      - name: Check Tag
+        id: check-tag
+        run: |
+          if [[ ${{ github.event.ref }} =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo ::set-output name=is_release::true
+            echo "is_release: true"
+          fi
```

```suggestion
    steps:
    # Determine whether this is a release based on the git tags that are applied.
    # Some other CI tasks check this and only run on releases.
      - name: Check Tag
        id: check-tag
        run: |
          if [[ ${{ github.event.ref }} =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
            echo ::set-output name=is_release::true
            echo "is_release: true"
          fi
```

Is my comment attempt correct?